lib/fastlane/actions/read_podspec.rb

```diff
@@ -0,0 +1,64 @@
+module Fastlane
+  module Actions
+    module SharedValues
+      READ_PODSPEC_JSON = :READ_PODSPEC_JSON
+    end
+
+    class ReadPodspecAction < Action
+      def self.run(params)
+        path = params[:path] || Dir['*.podspec*'].first
```

Comment:
Instead of automatically finding the file here, you can also use the `default_value` parameter in the `ConfigItem`:

```ruby
default_value: Dir['*.podspec*'].first
```

You could then even set `optional` to `false`, this way the user automatically gets asked for the missing value in case nothing was provided and it can't be found.